wallet/wallet.c

```diff
@@ -1132,11 +1132,11 @@ int wallet_extract_owned_outputs(struct wallet *w, const struct bitcoin_tx *tx,
 		utxo->blockheight = blockheight?blockheight:NULL;
 		utxo->spendheight = NULL;
 
-		log_debug(w->log, "Owning output %zu %"PRIu64" (%s) txid %s",
+		log_debug(w->log, "Owning output %zu %"PRIu64" (%s) txid %s %s",
```

Comment:
```suggestion
		log_debug(w->log, "Owning output %zu %"PRIu64" (%s) txid %s%s",
```

Moving the space between txid and state into the latter avoids trailing spaces :wink:

Reply:
:+1: